pkg/streamingpromql/testdata/upstream/functions.test

```diff
@@ -243,10 +243,9 @@ load 5m
 	http_requests_counter{path="/foo"} {{schema:0 sum:0 count:0 buckets:[0 0 0]}}+{{schema:0 sum:1 count:2 buckets:[1 1 1]}}x5
 	http_requests_mix{path="/foo"} 0 50 100 {{schema:0 sum:0 count:0 buckets:[0 0 0] counter_reset_hint:gauge}} {{schema:0 sum:1 count:2 buckets:[1 1 1] counter_reset_hint:gauge}}
 
-# Unsupported by streaming engine.
-# eval instant at 20m delta(http_requests[20m])
-# 	{path="/foo"} 200
-# 	{path="/bar"} -200
+eval instant at 20m delta(http_requests[20m])
+	{path="/foo"} 200
+	{path="/bar"} -200
 
 # Unsupported by streaming engine.
```

Comment:
Could you please enable the new `delta`-related tests below?

`tools/check-for-disabled-but-supported-mqe-test-cases` will help identify any test cases that might be able to be enabled.

Reply:
Oh missed that. Updated in e72c50452be2e0fce088f586bac246cec291814b